iron-flex-layout.html

```diff
@@ -259,6 +265,43 @@
       align-self: stretch;
     };
 
+    --layout-self-baseline: {
+      -ms-align-self: baseline;
+      /* currently not supported in safari */
+      -webkit-align-self: baseline;
+      align-self: baseline;
+    };
+
+    /* multi-line alignment in main axis */
+
+    /* currently not supported in ie */
+    /* currently not supported in safari */
```

Comment:
Well, this is not browser specific, it lacks implementation for some browsers... :) I suppose, it is not the only case in the whole codebase, but I let you judge of what should be integrated.